scss/upload/_layout.scss

```diff
@@ -210,8 +216,9 @@
     }
 
     .k-file-name-size-wrapper {
-        vertical-align: middle;
-        margin-left: 1em;
+        display: block;
+        margin-left: calc(#{$padding-x} + #{40px});
+        margin-right: calc(#{$padding-x} + #{16px + #{$padding-x-sm}});
```

Comment:
There are as follows:
-40px - width of the uploaded file image;
-16px - clear icon width